src/client/components/banner/banner.test.tsx

```diff
@@ -0,0 +1,30 @@
+import '@testing-library/jest-dom';
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Banner from './banner';
+
+describe('Banner', () => {
+  it('should render link if env var is defined', async () => {
+    // given
+    process.env.REACT_APP_EOP_BANNER_TEXT = 'This is a banner text'
```

Comment:
Thinking about it, we should probably cleanup `process.env` after each test. Something like
```ts
function withCleanupEnv() {
  const env = process.env;

  beforeEach(() => {
    process.env = { ...process.env };
  });
  
  afterEach(() => {
    process.env = env;
  });
}
```
That can then be used in all tests that mess with `process.env`.

Reply:
done